src/main/java/rx/internal/operators/SingleOperatorOnErrorResumeNextViaSingle.java

```diff
@@ -0,0 +1,67 @@
+package rx.internal.operators;
+
+import rx.Observable.Operator;
+import rx.Producer;
+import rx.Single;
+import rx.Subscriber;
+import rx.exceptions.Exceptions;
+import rx.plugins.RxJavaPlugins;
+
+public class SingleOperatorOnErrorResumeNextViaSingle<T> implements Operator<T, T> {
```

Comment:
Why not implement this via SingleOnSubscribe and then you have access to SingleSubscriber without the API mismatch with backpressured Operators?